src/lib/marquee.jsx

```diff
@@ -1,17 +1,32 @@
 import PropTypes from "prop-types"
-import { useState, useRef, useEffect, useCallback } from "react"
+import { useState, useMemo, useRef, useEffect, useCallback } from "react"
 
 import { classnames } from "./utils/classnames"
 
 import "./marquee.scss"
 
-export const Marquee = ({
+/**
+ * Infinite scrolling marquee component.
+ *
+ * @param {object} props
+ * @param {number=50} props.speed - The speed of the marquee in pixels per second.
+ * @param {number=20} props.reducedMotionSpeed - The speed of the marquee in pixels per second when prefersReducedMotion is true.
+ * @param {boolean=false} props.prefersReducedMotion - If true, marquee will use reducedMotionSpeed instead of speed.
+ * @param {boolean=true} props.play - If false, the marquee will not animate.
+ * @param {string} props.className - Additional classnames to add to the marquee.
+ * @param {React.Component} props.children - The content to be rendered inside the marquee. Will be wrapped in a div with the class of `marquee__slide`.
+ *
+ * @returns {React.Component}
+ */
+export function Marquee({
   speed = 50,
   reducedMotionSpeed = 20,
   prefersReducedMotion = false,
+  play = true,
```

Comment:
Let's update this prop name to `playing`, that way it's just a little bit less ambiguous that it's a boolean